tools/serverpod_cli/bin/create/create.dart

```diff
@@ -25,6 +25,16 @@ Future<void> performCreate(
   String template,
   bool force,
 ) async {
+  // Check whether the project name is valid or not
+  for (int i = 0; i < name.length; i++) {
+    int ascii = name[i].codeUnitAt(0);
+    if (!((ascii >= 97 && ascii <= 122) || name[i] == '_')) {
+      print(
+          "Project Name can only contain lowercase letters and underscores..");
```

Comment:
Please use single quotes for strings. Also, captialization and punctuation is incorrect. A better sentence would be:
`'Failed to create project. The project name may only contain lower case letters, numbers, and underscores.'`